.docs/user-guide/installation.md

```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
 curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable
 ```
 
+#### Install a specific version
+
+You can forcibly install a particular version like this:
+
+```sh
+$ curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable 0.7.0
+```
+
+The version shows during an install, but to see it again later:
```

Comment:
I do not agree that this suggested rewording is "cleaner". And I did an A-B compare using  online readability analyzers. My version returned significantly better scores.

I used this: https://readable.io/text/
And this: https://www.webpagefx.com/tools/read-able/check.php